set.mm

```diff
@@ -756151,6 +756179,534 @@ last digit of the integer part (for ` b = ` 10 the first digit before
       ZVIVBEZAIUIUJHEZVJDEVPVOUMPVNVQVFVIIVGUKRVFVRVNVDVRVEAUNOOMAVIUOUPSVNVKQE
       VFVNVKVNMVIMDEVNVCPVIVGUQURSRUSUTVA $.
   $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  The Ackermann function
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  According to Wikipedia ("Ackermann function", 8-May-2024,
+  ~ https://en.wikipedia.org/wiki/Ackermann_function ): "In computability
+  theory, the Ackermann function, named after Wilhelm Ackermann, is one of the
+  simplest and earliest-discovered examples of a total computable function that
+  is not primitive recursive. ... One common version is the two-argument
+  Ackermann-P&eacute;ter function developed by R&oacute;zsa P&eacute;ter and
+  Raphael Robinson.  Its value grows very rapidly; for example, ` A ( 4 , 2 ) `
+  results in ` 2 ^ ; 6 5 5 3 6 - 3 `, an integer of 19,729 decimal digits."
```

Comment:
> Finally, this statement calls for a proof, but I guess it would really be tedious ;-)

Alexander can correct me, but I actually think it's not that tedious. `A(4, 2) = A(3, A(4, 0)) = A(3, A(3, 1))` and this pull request already finds the value of `A(3, 1)` and gives a formula for `A(3, n)`. The power `2 ^ ; 1 6` which appears in the exponent has been already computed in Metamath theorem `2exp16`.

Reply:
Yes, i am just about to prove this and provide it with a separate PR. But since it is mentioned here, I will add it to this PR.

Reply:
Done